### ISSUE TITLE
Re-add "Custom Amounts"

### ISSUE THREAD

**Issue by Carb0nNova** [deathraygames/civ-clicker#33]
In older versions, after a certain point, you were able to specify a custom amount via a text box.

Once you get to the point where you can buy 10,000+ buildings at a time, this would be a handy feature.

**Comment by Lukenickerson** [comment]
@Carb0nNova -- This can be done via controls in the settings. Does that work for you?

**Comment by DeadlyKitten37** [comment]
No, It doesn't work well due to needing to go to the settings each and enable it each time you want to go add a larger amount.
It would be better to have 3 check boxes. 1 for the +1,+10,+100,+1k. a 2nd for the +Custom. a 3rd for the +infinity. That way It would be possible to chose better which buttons you need.

the fact that the +custom and +Infinity never appear together makes it quite hard to work with

**Comment by Lukenickerson** [comment]
I like that solution @DeadlyKitten37 . Do you think the buttons need to be customized per resources or just once (in the settings) for all resources?

**Comment by DeadlyKitten37** [comment]
I think just once is fine
